subsys/bluetooth/host/mesh/transport.h

```diff
@@ -26,6 +26,11 @@
 #define TRANS_CTL_OP_FRIEND_SUB_CFM    0x09
 #define TRANS_CTL_OP_HEARTBEAT         0x0a
 
+#define TRANS_CTL_OP_RREQ 						 0x0b
+#define TRANS_CTL_OP_RREP							 0x0c
+#define TRANS_CTL_OP_RWAIT						 0x0d
+#define TRANS_CTL_OP_RERR							 0x0e
```

Comment:
Do you have a reference to the specification where these values are defined? The Bluetooth SIG Mesh Profile Specification 1.0 (table 3.39) defines the range 0x0B–0x7F as "Reserved for Future Use". Simply because of this I don't think we would be able to include this code as part of the stack.

Reply:
It is possible to change the code outside of the range reserved for future use. We can use any thing above 0x7F I guess.

Reply:
Nope, there are only 7 bits reserved for the control codes in the Lower Transport PDU encoding, so 0x7f is the maximum. I can think of two possible ways to make do this in a clean (future-proof) way:

 - Work together with the Mesh Working Group from the Bluetooth SIG to formally allocate the control codes for your use case.

or

 - Implement this feature by only utilizing the model layer, i.e. don't touch any of the layers below it.